neurom/io/tests/test_utils.py

```diff
@@ -92,21 +93,53 @@ def _check_neurites_have_no_parent(nrn):
         nt.assert_true(n.root_node.parent is None)
 
 
+def test_get_morph_files():
+    ref = set(['Neuron_h5v2.h5', 'Neuron_2_branch_h5v2.h5', 'Neuron_slice.h5',
+               'Neuron.swc', 'Neuron_h5v1.h5', 'Neuron_2_branch_h5v1.h5'])
+
+    FILE_PATH = os.path.abspath(os.path.join(DATA_PATH, 'valid_set'))
+    files = set(os.path.basename(f) for f in utils.get_morph_files(FILE_PATH))
+
+    nt.assert_equal(ref, files)
+
+
```

Comment:
I have simply moved this function above test_load_neurons so that it is tested first. It is not a new code.

Reply:
ok